apps/.eslintignore

```diff
@@ -1,8 +1,8 @@
-src/**/*.min.js*
-src/calc/js-numbers/js-numbers.js
-src/code-studio/vendor/**/*.js
-src/third-party/**/*.js
+/src/**/*.min.js*
+/src/calc/js-numbers/js-numbers.js
+/src/code-studio/vendor/**/*.js
+/src/third-party/**/*.js
 
-lib
-build
+/lib
+/build
```

Comment:
Leading slash in this case means "relative to the .eslintignore file."  ESLint [uses gitignore format](http://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories). From the [gitignore documentation](https://git-scm.com/docs/gitignore): 

> A leading slash matches the beginning of the pathname. For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".